optimum/habana/transformers/generation/utils.py

```diff
@@ -678,6 +681,7 @@ def generate(
         model_kwargs["attn_softmax_bf16"] = generation_config.attn_softmax_bf16
 
         # determine whether limit_hpu_graphs needs to be used
+        model_kwargs["use_hpu_graphs"] = hpu_graphs
```

Comment:
Is it still needed after clear_cache removed?

Reply:
Lets have it ; no harm as we may need to use it in future.